### PR TITLE
feat(auth): added multi-language support to logIn method

### LINKED ISSUE #835: Login window closes when phrase "You have signed in." is shown - only works for English browser setting
## Expected behaviour
*When a user logs on the pop-up should be closed when the login process is completed*

## Current behaviour
*The login pop-up closes only when the browser language is set to English, for any other language the code can not find the phrase "You have signed in."*

## Environment info
**Client tech stack**: *VanillaJS, NodeJS etc.*
**Server type**: SASJS|SASVIYA|SAS9
**Login mechanism**: Redirected
**Debug**: false

In https://[cdn.jsdelivr.net/npm/@sasjs/adapter@4/index.min.js](https://cdn.jsdelivr.net/npm/@sasjs/adapter@4/index.min.js) 
e.g. _a.includes("You have signed in.")_ and _t.window.document.body.innerText.includes("You have signed in.")_

### PR DESCRIPTION
**Pull request by YuryShkoda**
## Issue

Closes #835 

## Intent

- Add multi-language support to `logIn` method of the `AuthManager`.

## Implementation

- Added `getExpectedLogInSuccessHeader` function to `src/auth/loginHeader.ts`.
- Added `isLogInSuccessHeaderPresent` function to `src/auth/loginHeader.ts`.
- Added `getUserLanguage` utility.

## Checks

No PR (that involves a non-trivial code change) should be merged, unless all items below are confirmed!  If an urgent fix is needed - use a tar file.

- [x] Unit tests coverage has been increased and a new threshold is set.
- [ ] All `sasjs-cli` unit tests are passing (`npm test`).
- (CI Runs this) All `sasjs-tests` are passing. If you want to run it manually (instructions available [here](https://github.com/sasjs/adapter/blob/master/sasjs-tests/README.md)).
- [ ] [Data Controller](https://datacontroller.io) builds and is functional on both SAS 9 and Viya

### CODE REVIEW

File: src/auth/AuthManager.ts [hunk 1]
```diff
@@ -14,6 +14,44 @@ export class AuthManager {
...
+    ru: `\u0412\u044b \u0432\u044b\u043f\u043e\u043b\u043d\u0438\u043b\u0438 \u0432\u0445\u043e\u0434 \u0432 \u0441\u0438\u0441\u0442\u0435\u043c\u0443.`,
+    el: `\u0388\u03c7\u03b5\u03c4\u03b5 \u03c3\u03c5\u03bd\u03b4\u03b5\u03b8\u03b5\u03af.`,
+    hr: `Prijavili ste se.`,
+    da: `Du er logget p\u00e5.`,
+    de: `Sie sind jetzt angemeldet.`,
+    sh: `Prijavljeni ste.`,
+    pt: `Iniciou sess\u00e3o.`,
+    hu: `Bejelentkezett.`,
+    sr: `Prijavljeni ste.`,
+    en: enLoginSuccessHeader,
```
Comment (medjedovicm) [inline src/auth/AuthManager.ts:54]: I see that `enLoginSuccessHeader` is used in multiple files (check references). I suppose this list should be used in those places as well. Also would be nice to have a single reusable source of this list, in the project.

Example:
image

File: src/auth/verifySas9Login.ts [hunk 2]
```diff
@@ -12,7 +13,7 @@ export async function verifySas9Login(loginPopup: Window): Promise<{
 
     isLoggedIn =
       loginPopup.window.location.href.includes('SASLogon') &&
-      loginPopup.window.document.body.innerText.includes('You have signed in.')
+      loginPopup.window.document.body.innerText.includes(enLoginSuccessHeader)
```
Comment (medjedovicm) [inline src/auth/verifySas9Login.ts:16]: Why did we hardcoded `EN`glish?

File: src/auth/verifySasViyaLogin.ts [hunk 3]
```diff
@@ -20,9 +21,7 @@ export async function verifySasViyaLogin(loginPopup: Window): Promise<{
     if (loginPopup.closed) break
     isAuthorized =
       loginPopup.window.location.href.includes('SASLogon') ||
-      loginPopup.window.document.body?.innerText?.includes(
-        'You have signed in.'
-      )
+      loginPopup.window.document.body?.innerText?.includes(enLoginSuccessHeader)
```
Comment (medjedovicm) [inline src/auth/verifySasViyaLogin.ts:24]: Why did we hardcoded `EN`glish?

**Review by medjedovicm — CHANGES_REQUESTED** [review]

**Review by allanbowe — APPROVED** [review]

**Comment by gh4nh** [comment]
@allanbowe I've downloaded and implemented the index.min.js and successfully tested with a German and English browser. Thank you for your speedy fix.

**Pull request merged**